packages/@atjson/document/src/index.ts

```diff
@@ -212,7 +212,11 @@ export default class Document {
   slice(start: number, end: number): Document {
     let DocumentClass = this.constructor as typeof Document;
     let slicedAnnotations = this.where(a => {
-      return a.start <= start && a.end >= start;
+      if (start < a.start) {
+        return start <= a.start && end > a.start;
```

Comment:
you don't need the first half of this:
`return end > a.start`

Reply:
yeah, but it's nice from a readability perspective to have that symmetry with the other branch